src/consumption.cpp

```diff
@@ -1260,7 +1265,6 @@ void Character::modify_morale( item &food, const int nutr )
             add_morale( MORALE_CANNIBAL, -60, -400, 60_minutes, 30_minutes );
         } else if( numb ) {
             add_msg_if_player( m_bad, _( "You find this meal distasteful, but necessary." ) );
-            add_morale( MORALE_CANNIBAL, -60, -400, 60_minutes, 30_minutes );
```

Comment:
This changes how numb works which seems out of the scope of this PR.

Reply:
Ah, yes, that was not my intention. I was confused why you said that "NUMB still considers cannibalism bad"; that's why I tried to change it.